testing/ui-tests/src/lib.rs

```diff
@@ -50,4 +50,5 @@ fn ui_tests() {
 fn ui_fail() {
     let t = trybuild::TestCases::new();
     t.compile_fail("src/incorrect/*.rs");
+    t.pass("src/correct/*.rs");
```

Comment:
Nit: Should we move this to the `ui_tests` category? Having it under `ui_fail` might be misleading

Reply:
or just move this one to `ui_tests` and keep `ui_fall`

Reply:
Ah, didn't spot the passing cases above; will move it